SgmlTests/Tests.cs

```diff
@@ -444,6 +443,68 @@ public void Test_XPathDocument_NameTable()
             Assert.AreEqual("abcd", xpathNavigator.Evaluate("string(/html/body)"), "Expecting '/html/body' to be evaluated correctly.");
 
         }
-#endif
+
+        /// <summary>This is a unit-test for the <see cref="SgmlReader.TextWhitespace"/> property's logic.</summary>
+        [Test]
+        public void Invalid_TextWhitespace_enum_values_should_be_dropped()
+        {
+            SgmlReader sgmlReader = new SgmlReader()
+            {
+                TextWhitespace = (TextWhitespaceHandling)0xFF
+            };
+
+            Assert.AreEqual(TextWhitespaceHandling.TrimBoth | TextWhitespaceHandling.OnlyLineBreaks, sgmlReader.TextWhitespace, "The " + nameof(sgmlReader.TextWhitespace) + " property should only respect defined flags bits.");
+        }
+
+        /// <summary>https://github.com/lovettchris/SgmlReader/issues/15</summary>
+        [Test]
+        public void DoBradescoTrailingWhitespaceTest()
```

Comment:
I couldn't think of a better name for the test

Reply:
Yeah, who is "Bradesco" ?

Reply:
[_Banco Bradesco_](https://banco.bradesco/) is the Spanish bank that generated the original OFX file used in OfxSharp's tests where I grabbed it from to demonstrate the whitespace-handling issue that sparked this whole PR: https://github.com/kevencarneiro/OFXSharp/blob/master/source/OfxSharp.NETCore.Tests/Files/bradesco.ofx

Reply:
Got it, perfect name then, leave it as is!